subworkflows/local/align.nf

```diff
@@ -13,7 +13,6 @@ include {   MUSCLE5_SUPER5                    } from '../../modules/nf-core/musc
 
 // Include local modules
 include {   CREATE_TCOFFEETEMPLATE            } from '../../modules/local/create_tcoffee_template' 
-include {   MTMALIGN_ALIGN                    } from '../../modules/local/mtmalign_align'
```

Comment:
Why this one is dropped?

Reply:
Because in the current version is not actually supported - Leon is working on that, so whenever it is ready we can add it back with the right nf-core import :)

Reply:
Fair enough, did not realized it was a local one